lib/Doctrine/Sequence.php

```diff
@@ -37,12 +37,12 @@ class Doctrine_Sequence extends Doctrine_Connection_Module
      * Returns the next free id of a sequence
      *
      * @param string $seqName   name of the sequence
-     * @param bool              when true missing sequences are automatic created
+     * @param bool    $onDemand          when true missing sequences are automatic created
      *
      * @return integer          next id in the given sequence
      * @throws Doctrine_Sequence_Exception
      */
-    public function nextId($seqName, $ondemand = true)
+    public function nextId($seqName, $onDemand = true)
```

Comment:
Just changing to match case of other instances I've seen of this (doesn't do anything here since the params aren't used).